libsolidity/ast/AST.h

```diff
@@ -509,6 +509,8 @@ class ContractDefinition: public Declaration, public StructurallyDocumented, pub
 	/// as intended for use by the ABI.
 	std::map<util::FixedHash<4>, FunctionTypePointer> interfaceFunctions(bool _includeInheritedFunctions = true) const;
 	std::vector<std::pair<util::FixedHash<4>, FunctionTypePointer>> const& interfaceFunctionList(bool _includeInheritedFunctions = true) const;
+	/// @returns the EIP-165 compatible interface identifier.
+	uint64_t interfaceId(bool _includeInheritedFunctions = false) const;
```

Comment:
Since the bool is unused, probably better to just remove it and leave a comment.

Reply:
agree

Reply:
Done.